code/obj/item/clothing/masks.dm

```diff
@@ -293,6 +293,23 @@
 	w_class = W_CLASS_SMALL
 	var/mob/living/carbon/human/victim
 
+	var/spam_flag = 0
+	var/spam_timer = 100
+	var/list/sounds_instrument = list('sound/musical_instruments/Bikehorn_1.ogg')
+	var/volume = 50
```

Comment:
why this useless var

Reply:
Rewrote this and the original mask to match.